### PR TITLE
Update cpp-stl-cheatsheet.md with pair sorting 

### PR DESCRIPTION
**Pull request by LegendaryGene**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by LegendaryGene** [comment]
Please Merge this so I can get the HacktoberFest submission @crescentpartha

File: CheatSheets/cpp-stl-cheatsheet.md [hunk 1]
```diff
@@ -276,7 +276,21 @@
...
+    // Vector pair sorting according to second element
+    cout << "Sorting according to second element\n";
+    bool second_sorter (const pair <int,int> &a, const pair <int,int> &b) //comparator function, to be put outside main
+    {
+    	return (a.second < b.second);
+    }
+    sort (v.begin(), v.end(), second_sorter);
+    cout << "\n";
+;```
 **[🔼Back to Top](#table-of-contents)**
```
Comment (crescentpartha) [inline CheatSheets/cpp-stl-cheatsheet.md:294]:
```suggestion
```
Remove this line from the block code and then add the **[🔼Back to Top](#table-of-contents)** below the block code.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Request changes.

**Comment by LegendaryGene** [comment]
Done, now please do @crescentpartha

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**